.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -26,6 +26,10 @@ Example: Ubuntu 16.04
 
 ### Logs
 
+```
```

Comment:
Maybe move this under the text at L33?

Reply:
For me it would make sense to first point out where to put it, as eventually they will read it under where to find it. But if it makes more sense for you I can do that, sure.